decisions/wc_dark_cult_decisions.txt

```diff
@@ -448,7 +448,11 @@ decisions = {
 				text = pick_main_deity_timer_tooltip
 				NOT = { has_character_modifier = picked_main_deity_timer }
 			}
-
+			conditional_tooltip = {
+				trigger = { has_character_flag = important_old_god_task_flag }
+				text = important_old_god_task_tooltip
+			}
+			hidden_tooltip = { NOT = { has_character_flag = important_old_god_task_flag } }
```

Comment:
Tooltip not working.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/82752450-2acb9e80-9dcf-11ea-9d1b-e24653475108.png)

</details>

```suggestion
			custom_tooltip = {
				text = important_old_god_task_tooltip
				NOT = { has_character_flag = important_old_god_task_flag }
			}
```